src/Postnl.php

```diff
@@ -324,6 +324,29 @@ public function generateLabelWithoutConfirm(ComplexTypes\Shipment $shipment, $pr
     }
 
     /**
+     * @param ComplexTypes\Shipment $shipment
+     * @param string $contactPerson
+     * @param string $email
+     * @return ComplexTypes\GenerateReturnLabelResponse
+     *
+     * @see LabellingClient::generateLabel()
+     */
+    public function generateReturnLabel(
+        ComplexTypes\Shipment $shipment,
+        $contactPerson,
+        $email
+    ) {
+        // Prepare arguments.
+        $message = new ComplexTypes\Message();
+        $customer = new ComplexTypes\Customer($this->customerNumber, $this->customerCode, $this->collectionLocation,
+            $contactPerson, $email, $this->customerName);
```

Comment:
eww, if a function call is multi-line, do it properly.

Just put every parameter on its own line, or better yet, use the factory + setters (one per line)